app/models/katello/errata_status.rb

```diff
@@ -33,17 +50,21 @@ def to_global(_options = {})
         ::HostStatus::Global::WARN
       when UP_TO_DATE
         ::HostStatus::Global::OK
-      when UNKNOWN
+      when NO_PACKAGES
+        ::HostStatus::Global::WARN
+      when NO_REPOSITORIES
+        ::HostStatus::Global::WARN
+      when UNREGISTERED
         ::HostStatus::Global::WARN
       else
         ::HostStatus::Global::WARN
       end
     end
 
     def to_status(_options = {})
-      return UNKNOWN if host.content_facet.nil?
+      return UNREGISTERED if host.content_facet.nil?
```

Comment:
@jlsherrill When the content facet has no UUID this status is considered irrelevant so it won't be shown in the UI. I think that makes this check redundant and therefore the UNKNOWN/UNREGISTERED status unnecessary. Am I not thinking of some (edge)case?

Reply:
As long as it also doesn't affect the global status in that case, i think you are correct